README.md

```diff
@@ -22,6 +22,32 @@ This will add the helper module to your `package.json` file as a development dep
 
 Both [Mocha](https://mochajs.org/) and [Should](https://shouldjs.github.io/) will be pulled in with the test helper.  Mocha is a unit test framework for Javascript; Should is an assertion library.  For more information on these frameworks, see their associated documentation.
 
+## Alternate linking of node project dependencies
```

Comment:
I think we should avoid recommending this setup, as it's prone to weirdness depending on whatever version of npm is running.  It's best practice to just add the dev dependencies, as this makes it easy to run the tests both in local development environments and CI (like Travis-CI).  Linking is only really useful when you are actively developing a dependency.  

* * *

Example: my node is `node-red-contrib-foobar`, and it needs dependency `foobar`. 
 But `foobar` has a bug affecting my NR Node, and I want to fix it. I clone `foobar` from GitHub, make my changes, run `npm link` in the working copy.  Then, I return to my `node-red-contrib-foobar` working copy and run `npm link foobar`.  My bug fix will then be reflected in `node-red-contrib-foobar`.  If I make *more* changes to `foobar`, `node-red-contrib-foobar` automatically gets them, because of the link.